style_guides/html_style_guide.md

```diff
@@ -32,53 +32,3 @@ It's important that when you write CSS selectors using classes, IDs, and attribu
 (keeping in mind that we should _never_ use IDs and attributes in our selectors), that the
 capitalization in the CSS matches that used in the HTML. [HTML and CSS follow different case sensitivity rules](http://reference.sitepoint.com/css/casesensitivity), and we can avoid subtle gotchas by ensuring we use the
 same capitalization in both of them.
-
-## Multiple attribute values
```

Comment:
:information_source: We're not really having any `HTML` files outside React anymore, and this rule and the next are covered by Prettier.